src/nuxt.ts

```diff
@@ -38,22 +37,18 @@ const resolveRootDir = () => {
 export async function loadFixture () {
   const { options } = getContext()
 
-  options.rootDir = resolveRootDir()
+  const rootDir = options.rootDir || resolveRootDir()
```

Comment:
We still should warn users if rootDir is not a valid nuxt app. What's the point of this change?